README.md

```diff
@@ -77,7 +76,7 @@ Example command:
 
 Install this project with:
 
-- cd opp-school-library
+- cd oop-school-library
```

Comment:
Nice job so far. let's make some final touches to make everything perfect
- [ ] your issue link under the contributing section in the README is broken. Please kindly add the correct link to make it professional
- [ ] the Email link you added under the author section is broken as well. Please add a correct one

Reply:
@tufoinnkuo10 Sure. Let me do that